.envs/.production/.postgres.tmpl

```diff
@@ -0,0 +1,10 @@
+# PostgreSQL
+# ------------------------------------------------------------------------------
+POSTGRES_HOST=postgres
+POSTGRES_PORT=5432
+POSTGRES_DB=reprohack_hub
+# Postgres user should be an alpanumeric string
+POSTGRES_USER=$TMPL_POSTGRES_USER
+# Postgres password should be an alpanumeric string
+POSTGRES_PASSWORD=jvWaPUIpOt6NezC5BBzBQNRWtKL73fNkYxOunmMTdlbUT2lNil4pIWWHEGOrhG4p
```

Comment:
I assume this is not a used password?

Reply:
Doh!  Fixed in an amended, force-pushed commit to this branch.  I've also changed that password on the dev VM and in an Ansible Vault.